src/github.js

```diff
@@ -111,6 +111,10 @@ async function createPrComment(repoToken, markupData, updateCommentIfOneExists,
   }
 }
 
+async function createJobSummary(markupData) {
```

Comment:
Where is this getting called in case the Summary is desired?

Reply:
It's not a complete PR.  Just showing the concept.  Once I get a thumbs up that your team is ok with the additional, I will finish up the missing pieces.

I didn't want to waste any cycles on something the BC swat team didn't even want implemented.

Reply:
@bradyclifford I think we're fine with adding the summary to this action.  Is this something you wanted to continue with or should we close this one out if you'll be moving to the other action?

Reply:
We can close this PR. I am going to use the new action instead now.